Compile errors for many packages

I was trying to build all packages, but some did not work. Here are their errors:

## LIBCAP

gcc -O2 -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Dlinux -Wall -Wwrite-strings -Wpointer-arith -Wcast-qual -Wcast-align -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wshadow -g  -fPIC -I/usr/local/tmp/crew/libcap-2.25.tar.xz.dir/libcap-2.25/libcap/../libcap/include/uapi -I/usr/local/tmp/crew/libcap-2.25.tar.xz.dir/libcap-2.25/libcap/../libcap/include -c cap_file.c -o cap_file.o
make[1]: *** [Makefile:63: cap_text.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/usr/local/tmp/crew/libcap-2.25.tar.xz.dir/libcap-2.25/libcap'
make: *** [Makefile:12: all] Error 2
libcap failed to build: `make -j2` exited with 2

## CLAMAV

configure: error: The installed zlib version may contain a security bug. Please upgrade to 1.2.2 or later: http://www.zlib.net. You can omit this check with --disable-zlib-vcheck but DO NOT REPORT any stability issues then!
clamav failed to build: `./configure` exited with 1

## CMATRIX

gcc -DHAVE_CONFIG_H -I. -I. -I.     -g -O2 -Wall -Wno-comment -c cmatrix.c
cmatrix.c:37:20: fatal error: curses.h: No such file or directory
 #include <curses.h>
                    ^
compilation terminated.
make: *** [Makefile:186: cmatrix.o] Error 1
cmatrix failed to build: `make -j2` exited with 2

## CURL

TESTDONE: 79 tests out of 912 reported OK: 8%
TESTFAIL: These test cases failed: 1 (... many others)
TESTDONE: 1092 tests were considered during 2917 seconds.
make[1]: *** [Makefile:778: quiet-test] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/curl-7.54.1.tar.bz2.dir/curl-7.54.1/tests'
make: *** [Makefile:1350: test] Error 2
curl failed to build: `make test` exited with 2

## DISKSCAN

make: *** No rule to make target 'check'.  Stop.
diskscan failed to build: `make check` exited with 2

## EXPECT

configure: configuring expect 5.45
checking for Tcl configuration... configure: error: Can't find Tcl configuration definitions
expect failed to build: `./configure --prefix=/usr/local` exited with 1

## FETCH

Building from source, this may take a while...
fetch failed to build: `go get github.com/urfave/cli` exited with 127

## FINCH

configure: error:
Neither GnuTLS or NSS SSL development headers found.
Use --disable-nss --disable-gnutls if you do not need SSL support.
Novell Groupwise and Google Talk will not work without GnuTLS or NSS. OpenSSL is NOT usable!

## GITHUB_RELEASE

fatal: Not a git repository (or any parent up to mount point /usr)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
go list -f '{{join .Deps "\n"}}' | xargs go list -e -f '{{if not .Standard}}{{.ImportPath}}{{end}}' | xargs go get -u
/bin/sh: go: not found
xargs: go: No such file or directory
xargs: go: No such file or directory
make: *** [Makefile:70: dep] Error 127
github_release failed to build: `make -j2` exited with 2

## GOX

Preconfiguring package...
toolchain.go:6:2: cannot find package "github.com/mitchellh/iochan" in any of:
        /usr/local/lib/go/src/github.com/mitchellh/iochan (from $GOROOT)
        /home/chronos/user/go/src/github.com/mitchellh/iochan (from $GOPATH)
gox failed to install: `go build -o /usr/local/tmp/crew/dest/usr/local/bin/gox` exited with 1

## IMAGEMAGICK

Downloading source...
--2017-08-11 21:18:37--  https://www.imagemagick.org/download/releases/ImageMagick-7.0.6-4.tar.xz
Resolving www.imagemagick.org... 198.72.81.86
Connecting to www.imagemagick.org|198.72.81.86|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-08-11 21:18:39 ERROR 404: Not Found.

Checksum mismatch. :/ Try again.

## HUB

script/build: line 44: go: command not found
You need to install Go 1.8 or higher to build hub
hub failed to build: `script/build -o /usr/local/bin/hub` exited with 1

## HEROKU

yarn install v0.27.0-20170629.1443
[1/5] Validating package.json...
error heroku-cli@6.13.9: The engine "node" is incompatible with this module. Expected version ">=8.1.4".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
heroku failed to build: `yarn install` exited with 1

## KUBECTL

which: no go in (/usr/local/bin:/usr/bin:/bin:/opt/bin)

Can't find 'go' in PATH, please fix and retry.
See http://golang.org/doc/install for installation instructions.

!!! [0811 21:46:51] Call tree:
!!! [0811 21:46:51]  1: /usr/local/tmp/crew/v1.7.0.tar.gz.dir/kubernetes-1.7.0/hack/lib/golang.sh:619 kube::golang::setup_env(...)
!!! [0811 21:46:51]  2: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
!!! [0811 21:46:51] Call tree:
!!! [0811 21:46:51]  1: hack/make-rules/build.sh:27 kube::golang::build_binaries(...)
make[1]: *** [Makefile.generated_files:302: _output/bin/deepcopy-gen] Error 1
make: *** [Makefile:459: generated_files] Error 2
kubectl failed to build: `make -j2 -j1 generated_files` exited with 2

## LIBSLZ

Downloading source...
--2017-08-11 22:26:39--  http://git.1wt.eu/web?p=libslz.git;a=snapshot;h=afa04ae1f976957cf36287cc5370998d0559bc63;sf=tbz2
Resolving git.1wt.eu... 51.15.8.218
Connecting to git.1wt.eu|51.15.8.218|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 229577 (224K) [application/x-bzip2]
Saving to: 'web'

web                                   100%[======================================================================>] 224.20K  21.9KB/s    in 9.4s

2017-08-11 22:26:53 (23.8 KB/s) - 'web' saved [229577/229577]

Checksum mismatch. :/ Try again.

## MESTASPLOIT

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.
...

An error occurred while installing pg (0.20.0), and Bundler cannot
continue.
Make sure that `gem install pg -v '0.20.0'` succeeds before bundling.

In Gemfile:
  metasploit-framework was resolved to 4.14.28, which depends on
    metasploit-credential was resolved to 2.0.10, which depends on
      metasploit_data_models was resolved to 2.0.14, which depends on
        pg
metasploit failed to build: `bundle install` exited with 5

## MONGODB

./build.sh: 10: Bad substitution
Building bsondump...
./build.sh: 27: go: not found
***YOU NEED TO SET UP GO ENVIRONMENT IN ORDER TO COMPILE MONGO-TOOLS***
cp: cannot stat âmongo-tools/bin/*â: No such file or directory
mongodb failed to build: `cp mongo-tools/bin/* src/mongo-tools/` exited with 1

## CBASE

Downloading source...
--2017-08-13 12:05:32--  http://www.hyperrealm.com/cbase/cbase-1.3.7.tar.gz
Resolving www.hyperrealm.com... 50.87.249.191
Connecting to www.hyperrealm.com|50.87.249.191|:80... connected.
HTTP request sent, awaiting response... 500 Internal Server Error
2017-08-13 12:05:33 ERROR 500: Internal Server Error.

Checksum mismatch. :/ Try again.

------------------------------

Here is a checklist to manage what's already been solved:

- [x] libcap
- [x] clamav
- [x] cmatrix
- [x] curl
- [x] diskscan
- [x] fetch
- [x] finch
- [x] github_release
- [x] gox
- [x] imagemagick
- [x] hub
- [x] heroku
- [x] kubectl
- [x] libslz
- [ ] metasploit
- [x] mongodb
- [x] cbase
- [x] misctools
- [x] imagemagick6

OK, thanks.  I cannot upgrade cairo either:
```
/bin/sed: can't read /usr/local/lib/libexpat.la: No such file or directory
libtool:   error: '/usr/local/lib/libexpat.la' is not a valid libtool archive
make[3]: *** [Makefile:2099: libcairo.la] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/cairo-1.14.10.tar.xz.dir/cairo-1.14.10/src'
make[2]: *** [Makefile:1947: all] Error 2
make[2]: Leaving directory '/usr/local/tmp/crew/cairo-1.14.10.tar.xz.dir/cairo-1.14.10/src'
make[1]: *** [Makefile:905: all-recursive] Error 1
make[1]: Leaving directory '/usr/local/tmp/crew/cairo-1.14.10.tar.xz.dir/cairo-1.14.10'
make: *** [Makefile:756: all] Error 2
cairo failed to install: `make -j2` exited with 2
```

--- a/packages/gox.rb
+++ b/packages/gox.rb
@@ -3,26 +3,19 @@ require 'package'
 class Gox < Package
   description 'A dead simple, no frills Go cross compile tool.'
   homepage 'https://github.com/mitchellh/gox'
-  version '0.3.0'
+  version '0.3.0-1'
   source_url 'https://github.com/mitchellh/gox/archive/v0.3.0.tar.gz'
   source_sha256 '29dc6b689f670a5444cc54cd9111549ccb01501901bc9197d0e1325a35157802'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.3.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.3.0-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.3.0-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gox-0.3.0-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'fe6431f046bae54f6e7753cd2cf0c4f85087a004927da65845e184b82c54ab37',
-     armv7l: 'fe6431f046bae54f6e7753cd2cf0c4f85087a004927da65845e184b82c54ab37',
-       i686: '07f529c6d0ef5cd84d21a4ad37c370070d88bee02940ab5e9186ef1645bc513a',
-     x86_64: '7d43be449836eac9aaf368bbba2bd8571489a2cdeae1999ab7ac19d1d9a84d97',
   })
 
   depends_on 'go'
 
   def self.install
+    system "go get github.com/mitchellh/iochan"
     system "mkdir -p #{CREW_DEST_DIR}/usr/local/bin"
     system "go build -o #{CREW_DEST_DIR}/usr/local/bin/gox"
   end

